### DISCUSSION TITLE
GitHub Self Hosted runner new parameter for workflow job ID

### DISCUSSION THREAD

**Discussion by bryanrichter** [community/community#125236] [Actions]
### Select Topic Area

Product Feedback

### Body

Apologies if this isn't where this should be, I have a feature request for the GitHub Actions self hosted runner. Currently, machines are assigned to any available job at GitHubs whim. It would be nice, especially for ephemeral runners, to be able to pass in the workflow job ID that a machine should be running.

**Comment by crohr** [comment]
Ohhh yes, would make pinning an ephemeral runner to a specific job so much easier!
